portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,59 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Pablo's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body id="background"> 
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 id="header"><b>Pablo Dueñas' Portfolio:</b></h3>
```

Comment:
Line 11 opens an `<h1>` tag, but never closes it. Maybe you intended to write `</b></h1>`?